helper/staking/staking.go

```diff
@@ -110,18 +115,22 @@ var (
65b82525050565b6000610d8982610ff6565b610d93818561100e565b9350610d9e83610fe6565b8060005b83811015610dcf578151610db68882610d48565b9750610dc183611001565b925050600181019050610da2565b5085935050505092915050565b610de5816110cc565b82525050565b6000610df860448361101f565b9150610e03826111b0565b606082019050919050565b6000610e1b601d8361101f565b9150610e2682611225565b602082019050919050565b6000610e3e60278361101f565b9150610e498261124e565b604082019050919050565b6000610e6160128361101f565b9150610e6c8261129d565b602082019050919050565b6000610e84601a8361101f565b9150610e8f826112c6565b602082019050919050565b610ea3816110d8565b82525050565b610eb281611114565b82525050565b6000602082019050610ecd6000830184610d6f565b92915050565b60006020820190508181036000830152610eed8184610d7e565b905092915050565b6000602082019050610f0a6000830184610ddc565b92915050565b60006020820190508181036000830152610f2981610deb565b9050919050565b60006020820190508181036000830152610f4981610e0e565b9050919050565b60006020820190508181036000830152610f6981610e31565b9050919050565b60006020820190508181036000830152610f8981610e54565b9050919050565b60006020820190508181036000830152610fa981610e77565b9050919050565b6000602082019050610fc56000830184610e9a565b92915050565b6000602082019050610fe06000830184610ea9565b92915050565b6000819050602082019050919050565b600081519050919050565b6000602082019050919050565b600082825260208201905092915050565b600082825260208201905092915050565b600061103b82611114565b915061104683611114565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0382111561107b5761107a61111e565b5b828201905092915050565b600061109182611114565b915061109c83611114565b9250828210156110af576110ae61111e565b5b828203905092915050565b60006110c5826110f4565b9050919050565b60008115159050919050565b60006fffffffffffffffffffffffffffffffff82169050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b600080fd5b7f4e756d626572206f662076616c696461746f72732063616e2774206265206c6560008201527f7373207468616e204d696e696d756d52657175697265644e756d56616c69646160208201527f746f727300000000000000000000000000000000000000000000000000000000604082015250565b7f4f6e6c79207374616b65722063616e2063616c6c2066756e6374696f6e000000600082015250565b7f56616c696461746f72207365742068617320726561636865642066756c6c206360008201527f6170616369747900000000000000000000000000000000000000000000000000602082015250565b7f696e646578206f7574206f662072616e67650000000000000000000000000000600082015250565b7f4f6e6c7920454f412063616e2063616c6c2066756e6374696f6e000000000000600082015250565b6112f8816110ba565b811461130357600080fd5b50565b61130f81611114565b811461131a57600080fd5b5056fea2646970667358221220a8138d18f101e6f82542c730da59e80cf72514739559efba0f22714c63d74fa164736f6c63430008070033"
```

Comment:
This bytecode doesn't match the `deployedBytecode` from the staking SC PR ⚠️ 

Please update it 🙏

Reply:
Fixed in https://github.com/0xPolygon/polygon-edge/pull/397/commits/d27667f7f5f16452596e14659a9aa9a2f7552db0